lib/housing-available/housing-available.models.js

```diff
@@ -44,9 +36,32 @@ module.exports = {
     HousingAvailable.associate = (models) => {
       models.HousingAvailable.belongsTo(models.Account);
       models.HousingAvailable.belongsTo(models.Location);
-      models.HousingAvailable.belongsTo(models.Landlord);  
+      models.HousingAvailable.belongsTo(models.Landlord);
       models.HousingAvailable.belongsTo(models.Property);
     };
-    return HousingAvailable;  
+    return HousingAvailable;
   },
+
+  id: Joi.object().keys({
+    id: Joi.string().guid()
+  }),
+  // Define api;
+  api: Joi.object().keys({
```

Comment:
I was able to submit the payload "{}"; something in this payload should be required. I think at least contact, title, and body would be an acceptable start.